### PR TITLE
Disables Photon.

### LINKED ISSUE #32: Images are compressed vertically
In this blog post: http://www.chrisfinke.com/2015/09/28/and-some-benches/amp/ the images are squished to about 60px tall but are still the full width of the column.

### PR DESCRIPTION
**Pull request by kraftbj**
Photon strips height/width attr from the img tag, causing many problems with AMP.

Fixes #32. See #14

### CODE REVIEW

**Comment by mjangda** [comment]
Sweet. Still might be good for us to run the image itself through Photon when we convert to elements `amp-img` but let's run with this for now.

**Comment by kraftbj** [comment]
@mjangda Agreed. Right now, there isn't an easy way to give Photon a full WP url and return a fully Photon URL (e.g. example.com/wp-content/uploads/whatever-200x200.jpg and have it spit out i1.wp.com/example.com/wp-content/uploads/whatever.jpg?w=200&h=200 )

The closest function would convert the above example to i1.wp.com/example.com/wp-content/uploads/whatever-200x200.jpg which both eliminates the advantages of the single-cached-dynamically-resized image bit of Photon so not terribly crazy about that.

tl;dr: Totally see this as a bandaid to make AMP functional under Photon until a better solution is vetted and flushed out.
